docs/community/projects.md

```diff
@@ -19,6 +19,7 @@
 - [Ngxs Effects](https://github.com/hsdgit/State.Of.Angular/tree/effects)
 - [Reactive Programming with Observables & NGXS](https://github.com/kctang/reactive-with-ngxs)
 - [Port of NgRx Example App](https://github.com/eranshmil/ngxs-example-app)
+- [State Management with Ngxs](https://github.com/joaqcid/state-mgmt-with-ngxs)
```

Comment:
Change please to `NGXS` and I'll merge :)

Reply:
updated!